src/convertToObject.js

```diff
@@ -6,7 +6,11 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resut = {};
+
+  const array = sourceString.split(';');
+
+  
 }
```

Comment:
The function 'convertToObject' is currently empty and does not implement the required functionality. It should parse the 'sourceString', split it into individual declarations, and then map each declaration to a key-value pair in the resulting object.